CLR/Tools/PlatformDesigner/ComponentObjectModel/MsBuildWrapper.cs

```diff
@@ -3137,6 +3138,13 @@ internal void SaveProjectProj(MFProject mfproj)
 
                     ProjectPropertyGroupElement bpg = SaveStringProps(proj, mfproj, tbl);
 
+                    if (mfproj.Properties.Any(p => p.Name == "reducesize"))

```

Comment:
This appears to be enumerating the properties twice. Any reason not to just use FirstOrDefault() to get both cases with only one pass?